pom.xml

```diff
@@ -37,22 +37,21 @@
     <properties>
         <!-- Versioning -->
         <version.arquillian.core>1.1.11.Final</version.arquillian.core>
-        <version.com.opera>1.5</version.com.opera>
         <version.phantom.driver>1.2.1.Final</version.phantom.driver>
-        <version.selenium>2.53.1</version.selenium>
-        <version.htmlunit.driver>2.20</version.htmlunit.driver>
+        <version.selenium>3.0.1</version.selenium>
+        <version.htmlunit.driver>2.23</version.htmlunit.driver>
         <version.junit>4.11</version.junit>
```

Comment:
Yeah, I've noticed these two things. But then still, shouldn't we be relying on what particular `arq-core` ships? I guess that's what we do for `gradle` multi build. So this change in `pom.xml` won't affect it? Or does it take precedence? Imagine `junit:4.13` introduces some breaking changes (we already rely on deprecated methods in `arq-core` so it's more the question of when than if) - this build will have to take this into consideratiojn.